auth/src/main/resources/pnc-config.json

```diff
@@ -5,7 +5,10 @@
             "@module-config":"authentication-config",
             "username":"${env.PNC_EXT_OAUTH_USERNAME}",
             "password":"${env.PNC_EXT_OAUTH_PASSWORD}",
-            "baseAuthUrl":"${env.PNC_EXT_REST_BASE_URL}"
+            "auth-server-url":"${env.PNC_EXT_REST_BASE_URL}",
+            "realm-public-key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAjG+B9bN0bQNDzpazVrpIG5SLEGmz7iehC/IjKLkFhwetP+nIVQi5hn59osJNrQ/vTGDdEYWLProLzkhJHRQzwZs2ucvyolaL/FcQ+l7UCxrewyhSsIubra14dJwhglKqQS+ZD2nRcQuiMJ97J9D9gqnk8z31mUO1Oq57HSIEwq/VH8YEWsOSnvd1ZDoYiKffnrghq1DTaEgPZm3aKPMmPkAS3n7ClC2Doww8r18EOEh+y8cea0M3fOpFL81137likn1m0phSHmdJ4MLVhUwgFn50UDGTjzB3pFAkUWE7s473C4zxAamLA1740hnUgXoCL36KKaMbZLQ4ZJnQEZQL6wIDAQAB",
+            "resource": "pncdirect",
+            "realm": "pncredhat"
```

Comment:
@matejonnet AFAIK you need them to construct the initial request and token at https://github.com/project-ncl/pnc/blob/master/auth/src/main/java/org/jboss/pnc/auth/ExternalAuthentication.java#L87 or do you want them to be configurable via env properties as well?